transcom/mymove
# MAIN - B-19794 SC-ability-to-open-PPM-after-Closeout

## [B-19794](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19794)

INT PR: https://github.com/transcom/mymove/pull/12992

## Summary

This is to add the ability for SC to view the PPM Documents once the Closeout SC has already reviewed it.
All the items in while viewing documents will be in read-only mode and fields will be disabled.

### How to test

1. Use `MoveWithPPMShipmentReadyForFinalCloseout` testharness to create a move.
2. As a customer -> go to the move and complete uploading PPM documents.
3. As a closeout SC -> find the move and click "Review Documents".
4. Accept or deny any items and complete/confirm the review.
5. You should now see "View Documents" button instead of "Review Documents".
6. Click on "View Documents" and you should be able to see all the fields same as Review Documents, but all the fields disabled.

## Screenshots
![image](https://github.com/transcom/mymove/assets/146856854/c32a2978-3c4f-4f9c-bdee-0056d86914e8)
![image](https://github.com/transcom/mymove/assets/146856854/57672ff2-265b-4bdd-867a-773ca9751af4)
